### PR TITLE
Add css, html in main and in app-routing.module.ts insert import and path.

### PR DESCRIPTION
**Pull request by Alexshailiha**
Create const routes for account, add-recipe, login, main, and sign-up. Delete the main front page on local server 4200 created by Angular. Frontend works on the terminal after writing: cd frontend/help-chef 
ng serve 
It is on local server: 4200.

### CODE REVIEW

**Comment by Alexshailiha** [comment]
image

File: frontend/help-chef/src/app/app.module.ts [hunk 1]
```diff
@@ -1,13 +1,15 @@
 import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
+import { ReactiveFormsModule } from '@angular/forms';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
-import { MainComponent } from './app/main/main.component';
```
Comment (Alexshailiha) [inline frontend/help-chef/src/app/app.module.ts:6]: One mistake and localhost isn't working ;)

Comment (adamkowalsky11) [reply frontend/help-chef/src/app/app.module.ts:6]: Great that you found the solution for the issue :D

**Review by adamkowalsky11 — APPROVED** [review]

**Comment by kmachura** [comment]
good job, Ola! well done!

**Review by kmachura — APPROVED** [review]
I am not an UI expert, looks good :)

**Review by wdubel — APPROVED** [review]
nice work

**Pull request merged**